openshift/tests-extension/test/olmv1-webhooks.go

```diff
@@ -0,0 +1,383 @@
+package test
```

Comment:
All tests here are for olmv1
We never will add tests that are not for olmv1
So, could we keep the orginal name of the file so that we can easily review the changes?
What is new or not?

Reply:
No, I just renamed the file to conform with the others.

Reply:
I don't mind either way. It would just be good to have consistent naming of files in the package. I'll do a git mv.

Reply:
I've reverted for now. Maybe we can do a separate PR to rename everything one way or the other.